lib/install.js

```diff
@@ -213,7 +215,31 @@ function install(opts, cb) {
     );
     fs.createReadStream(gzipFilePath).pipe(gunzip).pipe(writeStream)
       .once('error', cb.bind(null, new Error('Could not read ' + gzipFilePath)))
-      .once('finish', function() { cb() })
+      .once('finish', function() {
+        if (from.substr(-7) === '.tar.gz') {
+          var destDir = path.join(os.tmpdir(), 'selenium-standalone-' + Math.random());
```

Comment:
Can we do all in-memory instead of writing to disk? Maybe there's a module to handle this.